microsoft-edge/devtools-guide-chromium/resources/index.md

```diff
@@ -5,7 +5,7 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
```

Comment:
Change description yaml field from:
description: Organize resources by frame, domain, type, or other criteria.
to:
description: Browse and inspect the resource files that make up a webpage, and filter them by filetype, frame, domain, or other criteria.

Reply:
done in local copy